Gems/AtomLyIntegration/CommonFeatures/Code/Source/PostProcess/EditorModeFeedback/EditorEditorModeFeedbackSystemComponent.cpp

```diff
@@ -0,0 +1,268 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
```

Comment:
nit: Is the 'EditorEditor..Component' repeating Editor on purpose? perhaps choosing better naming for clarity (Manager..?)

Reply:
+1, feels like the first 'Editor' can be removed 😛

Reply:
Going over the class code, I now see that this is not a system component' as per LY/Atom (correct me if I'm wrong), In this case I recommend changing the name for both clarity and avoid misguiding.  Perhaps name along the line of 'EditorModeFeedbackSystemManagerComponent'.
I'm not inlove with the render code inside an editor component since in a way this almost feels to me like a feature processor that collects data and prepares it for the passes before the render - maybe an option to think about.
Another option is that logic part can be moved to the Controller class, hence using Adapter / Controller / config / Component scheme, but I still don't like render code ending up in such an outer layer.
@VickyAtAZ, what do you think?